### ISSUE TITLE
fine-tuning

### ISSUE THREAD

**Issue by my-221** [AGI-Edgerunners/LLM-Adapters#80]
May I ask, what was the data volume during your fine-tuning process? How many training rounds did you conduct? Is it necessary for the loss to converge?
